src/components/pages/ProjectInfo/OrganizationsList.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types'
```

Comment:
I just moved this out of the `ProjectInfo` main file as it was noisy. Its pure copy paste of old code.